api/src/main/java/org/openmrs/module/openconceptlab/client/OclClient.java

```diff
@@ -42,11 +45,16 @@
 import java.util.zip.ZipInputStream;
 
 public class OclClient {
-	
+
 	public static final String DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ss'Z'";
-	
+
 	public static final String FILE_NAME_FORMAT = "yyyyMMdd_HHmmss";
-	
+
+	public static final String oclApiUrl = "api.openconceptlab.org";
+
+	//if number is five that means the subscription url contains the collection version
+	public static final int NUMBER_OF_FORWARD_SLASHES = 5;

```

Comment:
Can you give an example of what such a value would look like with the 5  forward slashes?

Reply:
https://api.openconceptlab.org/users/Jecihjoy/collections/NOC/v2.0

Reply:
Like in code or just here in the comments?

Reply:
Which one do you think would be better? 😊

Reply:
@dkayiwa  I'm still not sure about this, does it make sense to have a lot of comments in the code?

Reply:
What is your definition of `a lot`?

Reply:
Is this good enough?

		/*
		 *This checks if collection version has been passed to subscription url by checking number of forward slashes
		 *If the number is 5, such as with https://api.openconceptlab.org/users/username/collections/collectionname/v1.0
		 *that means collection version was passed and it's assigned to exportVersion
		 */